pokemon_list.txt

```diff
@@ -1,2 +1,152 @@
-Bulbasaur Ivysaur Venusaur Charmander Charmeleon Charizard Squirtle Wartortle Blastoise Caterpie Metapod Butterfree Weedle Kakuna Beedrill Pidgey Pidgeotto Pidgeot Rattata Raticate Spearow Fearow Ekans Arbok Pikachu Raichu Sandshrew Sandslash Nidoran♀ Nidorina Nidoqueen Nidoran♂ Nidorino Nidoking Clefairy Clefable Vulpix Ninetales Jigglypuff Wigglytuff Zubat Golbat Oddish Gloom Vileplume Paras Parasect Venonat Venomoth Diglett Dugtrio Meowth Persian Psyduck Golduck Mankey Primeape Growlithe Arcanine Poliwag Poliwhirl Poliwrath Abra Kadabra Alakazam Machop Machoke Machamp Bellsprout Weepinbell Victreebel Tentacool Tentacruel Geodude Graveler Golem Ponyta Rapidash Slowpoke Slowbro Magnemite Magneton Farfetch'd Doduo Dodrio Seel Dewgong Grimer Muk Shellder Cloyster Gastly Haunter Gengar Onix Drowzee Hypno Krabby Kingler Voltorb Electrode Exeggcute Exeggutor Cubone Marowak Hitmonlee Hitmonchan Lickitung Koffing Weezing Rhyhorn Rhydon Chansey Tangela Kangaskhan Horsea Seadra Goldeen Seaking Staryu Starmie Mr. Mime Scyther Jynx Electabuzz Magmar Pinsir Tauros Magikarp Gyarados Lapras Ditto Eevee Vaporeon Jolteon Flareon Porygon Omanyte Omastar Kabuto Kabutops Aerodactyl Snorlax Articuno Zapdos Moltres Dratini Dragonair Dragonite Mewtwo Mew MissingNo.
-
+abra
+aerodactyl
+alakazam
+arbok
+arcanine
+articuno
+beedrill
+bellsprout
+blastoise
+bulbasaur
+butterfree
+caterpie
+chansey
+charizard
+charmander
+charmeleon
+clefable
+clefairy
+cloyster
+cubone
+dewgong
+diglett
+ditto
+dodrio
+doduo
+dragonair
+dragonite
+dratini
+drowzee
+dugtrio
+eevee
+ekans
+electabuzz
+electrode
+exeggcute
+exeggutor
+farfetch'd
+fearow
+flareon
+gastly
+gengar
+geodude
+gloom
+golbat
+goldeen
+golduck
+golem
+graveler
+grimer
+growlithe
+gyarados
+haunter
+hitmonchan
+hitmonlee
+horsea
+hypno
+ivysaur
+jigglypuff
+jolteon
+jynx
+kabuto
+kabutops
+kadabra
+kakuna
+kangaskhan
+kingler
+koffing
+krabby
+lapras
+lickitung
+machamp
+machoke
+machop
+magikarp
+magmar
+magnemite
+magneton
+mankey
+marowak
+meowth
+metapod
+mew
+mewtwo
+mr. mime
+missingno.
```

Comment:
is there a period at the end of this?  is that actually part of the name?

Reply:
Yes, it's part of the name. It's a "pokemon" that appears under specific circumstances and people used it to "hack" their original (Red/Yellow/Blue) games to duplicate items if I remember right.

Reply:
mhmm ok - that may get weird in the future depending on what you do.  `.` is a special character that has special meaning in lots of different things, so if you pass in this name verbatim it'll do weird things in regex, search, string splitting, etc.  Fine to keep it now - but watch out for it in the future.

Reply:
Ah, good point. It was mostly there for curiosity's sake but I'll just remove it since it's obviously not a "real" pokemon anyways.